VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/EstimatedDecisionDate/EstimatedDecisionDate.tsx

```diff
@@ -30,7 +30,7 @@ function EstimatedDecisionDate({ maxEstDate, showCovidMessage }: EstimatedDecisi
   //Opting to leave this code in place...because removing it will clearly kickstart pandemic part 4000.
   if (showCovidMessage) {
     return (
-      <AlertBox border="warning" text={t('claimDetails.covidMessage')}>
+      <AlertWithHaptics variant="warning" description={t('claimDetails.covidMessage')}>
         <Box mt={theme.dimensions.standardMarginBetween}>
           <Button
```

Comment:
Could potentially be simplified moving this to the `primaryButton` or `secondaryButton` props to AlertWithHaptics.